src/applications/personalization/dashboard/containers/DashboardAppWrapper.jsx

```diff
@@ -88,7 +96,7 @@ class DashboardAppWrapper extends React.Component {
         >
           <Breadcrumbs>{this.renderBreadcrumbs()}</Breadcrumbs>
 
-          {this.props.children}
+          <DashboardApp />
```

Comment:
DashboardAppWrapper is no longer used by react router so it no longer needs to render `children` that are passed to it. It just always renders the `DashboardApp`